### ISSUE TITLE
allow for a flag that executes immediately

### ISSUE THREAD

**Issue by Haroenv** [nvbn/thefuck#531]
I often do the same `git push` to a new brach that needs `--set-upstream origin ...` to create a new branch. `fuck` can fix this, but it forces me to press `enter` even though I know what input it needs. I _could_ get an alias that creates a branch immediately and don't require `fuck` at all anymore, but I'd prefer to be able to `fuck -y` that executes the suggested command immediately.

**Comment by scorphus** [comment]
Try with the following entry in `~/.config/thefuck/settings.py`:

```python
require_confirmation = False
```

**Comment by Haroenv** [comment]
But what if I want to do it by default with confirmation and just once without, so with a flag; would `fuck -y` be possible?

**Comment by scorphus** [comment]
Yes, at shell alias/function level.

**Comment by scorphus** [comment]
@Haroenv are you willing to code it? 😃  What shell do you commonly use?

**Comment by Haroenv** [comment]
I use Fish usually, and yes, I'll look at how this would work internally

**Comment by scorphus** [comment]
Awesome! I use Fish too! Please feel free to ask for help, although I'm sure you'll nail it :o)

**Comment by nvbn** [comment]
So I've checked a bit, and alias like:
```bash
alias fuck-it='export THEFUCK_REQUIRE_CONFIRMATION=False; fuck; export THEFUCK_REQUIRE_CONFIRMATION=True'
```
should work.

**Comment by Haroenv** [comment]
Thanks, and sorry for not following up on the PR

**Comment by nvbn** [comment]
After changes in #620 we can have `-y/--yes`:

```bash
➜ git br
git: 'br' is not a git command. See 'git --help'.

Did you mean one of these?
	branch
	var
➜  fuck -y
git branch
* 614-repeate-option
  620-command-line-arguments
  Asday-master
```

**Comment by nvbn** [comment]
Implemented in 3.16.
